lib/scripts/mtw.js

```diff
@@ -107,6 +103,14 @@ export class ContentScript {
       style.href  = chrome.extension.getURL('/assets/css/MTWStyles.css');
       document.getElementsByTagName('head')[0].appendChild(style);
       document.styleSheets[0].insertRule('span.mtwTranslatedWord {' + cssStyle + '}', 0);
+      /**
```

Comment:
@rohankatyal29 why are you adding them here? Look into `MTWStyles.css`. They are already there. Also if you want to add any CSS styles, add directly in that file.